README.md

```diff
@@ -141,7 +161,9 @@ Go to http://bin.mailgun.net. The Postbin will generate a special URL. Save that
 
 ```php
 # First, instantiate the SDK with your API credentials and define your domain. 
-$mg = new Mailgun('key-example', 'bin.mailgun.net', 'aecf68de', $ssl = False);
+$mg = new Mailgun('key-example', null, 'bin.mailgun.net');
+$mg->setApiVersion('aecf68de');
+$mg->setSslEnabled('false');
```

Comment:
Should probably have an example that is closer to how it's used with mailgun. What is the 'bin.mailgun.net'? 

Most users would do something similar:

```
$mg = new Mailgun('your-key');

....
```
And that should be it. That way SSL is enabled by default and less confusion.